modules/web/src/app/shared/model/NodeProviderConstants.ts

```diff
@@ -35,7 +35,12 @@ export enum NodeProvider {
   EDGE = 'edge',
   NONE = '',
 }
-
+export const PROVIDERS_SUPPORT_IPRANGES_NODEPORTS = [
```

Comment:
Perhaps this is better:
```suggestion
export const NODEPORTS_IPRANGES_SUPPORTED_PROVIDERS = [
```